### ISSUE TITLE
SimpleToggleButton issues

### ISSUE THREAD

**Issue by samreid** [phetsims/faradays-law#60]
While looking at keyboard accessibility, I came across SimpleToggleButton.  It looks like it should be replaced by sun/RadioButtonGroup.  (It has other problems related to its handling of options, but just getting rid of it and using RadioButtonGroup instead seems like the superior long term solution).

**Comment by jbphet** [comment]
Looks like this was taken care of back in April when instrumenting the sim, see 09174cf1371ef5c80f5f9bd70511fb802c6a7206.  Closing.
